template/src/api/middlewares/message-validator.js

```diff
@@ -0,0 +1,17 @@
+const AsyncApiValidator = require('asyncapi-validator');
+
+module.exports = (message, next) => {
+  if (message.headers.cmd === 'publish') {
+    validateMessage(message.payload, message.topic);
```

Comment:
I don't `message.topic` exists.
See these to check what properties exists: https://github.com/asyncapi/parser-js/blob/master/API.md#modules

You need `message.name`, `message.payload` and `channel` and `operation`.